subsys/bluetooth/controller/ll_sw/nordic/lll/lll_conn.c

```diff
@@ -50,6 +50,11 @@ static uint16_t trx_cnt;
 static uint8_t mic_state;
 #endif /* CONFIG_BT_CTLR_LE_ENC */
 
+#if defined(CONFIG_BT_CTLR_FORCE_MD_COUNT) && \
```

Comment:
as i am big fan of avoiding preprocessor i would do something like
```
#if defined(CONFIG_BT_CTLR_FORCE_MD_COUNT) && (CONFIG_BT_CTLR_FORCE_MD_COUNT > 0)
#define FORCE_MD_COUNT CONFIG_BT_CTLR_FORCE_MD_COUNT 
#else
#define FORCE_MD_COUNT 0
#endif
```
And use that everywhere else as C value.

Reply:
yes, will reduce the conditional compile noise.